micro-ui/web/packages/modules/core/src/components/Home.js

```
@@ -98,6 +99,87 @@ const CitizenHome = ({ getCitizenMenu, fetchedCitizen, isLoading }) => {
+          if (seenLinks.has(key)) return false;
+          seenLinks.add(key);
+          return true;
+        });
+
+      return (
+        <Card key={code} className="digit-landing-page-card" style={{ borderRadius: "1rem" }}>
+          <div className="icon-module-header">
+            <div className="digit-landingpagecard-icon">
+              <CustomSVG.Devices fill="#C84C0E" width="1rem" height="1rem" />
+            </div>
+            <div className="ladingcard-moduleName" role="heading" aria-level="2">
```

Biome (2.3.14)

[error] The elements with this role can be changed to the following elements:
<h1>
<h2>
<h3>
<h4>
<h5>
<h6>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)